src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.xml

```diff
@@ -31,6 +31,10 @@
             <tag name="cache.pool" />
         </service>
 
+        <service id="cache.security_expression_language" parent="cache.system" public="false">
```

Comment:
> public="false"

no need, that's already the default

Reply:
oups no sorry, having "parent", this is required!

Reply:
shouldn't this service be defined in SecurityBundle?

Reply:
Yes makes more sense :+1: 

And we need to require `symfony/framework-bundle` then to have `cache.system` available?

Reply:
Or we only use the cache if the parent service `cache.system` is available?

Reply:
framework-bundle is an implicit dep of any bundle, so no need

Reply:
and the fact that you reference `cache.security_expression_language` in SecurityBundle already creates this dependency anyway.

I agree about moving it to SecurityBundle